releasenotes/notes/fix-aqc-optimizer-typehint-34b54c6278d23f79.yaml

```diff
@@ -0,0 +1,21 @@
+---
+fixes:
```

Comment:
Given that the introduction of the alternative should have happened before, I am considering this a "bugfix", and showing the corresponding release note, but it is a bit of a tricky PR because I could also see it as being a new feature.